scipy/sparse/_data.py

```diff
@@ -206,19 +215,19 @@ def _min_or_max(self, axis, out, min_or_max):
             if self.nnz == 0:
                 return zero
```

Comment:
leaving this at return zero implies that the non-explicit elements have value zero...perhaps return None in this case?